src/fty-alert-flexible.cc

```diff
@@ -125,10 +126,13 @@ int main (int argc, char *argv [])
         log_error ("Failed to load config file %s",config_file);
     }
 
-    if (!streq(logConfigFile,""))
+    if (!streq(logConfigFile, ""))
```

Comment:
ensure logConfigFile default is FTY_COMMON_LOGGING_DEFAULT_CFG

Reply:
Take into account